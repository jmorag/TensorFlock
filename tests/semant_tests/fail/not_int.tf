main = 12;

wrong_type : Nat;
wrong_type = True;
