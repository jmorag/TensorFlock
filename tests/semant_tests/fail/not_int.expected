Fatal error: exception Failure("Declared type Nat but received type Bool")
